code/modules/antagonists/blob/overmind.dm

```diff
@@ -154,6 +154,7 @@ GLOBAL_LIST_EMPTY(blob_nodes)
 
 	if(announcement_time && (world.time >= announcement_time || blobs_legit.len >= announcement_size) && !has_announced)
 		priority_announce("Confirmed outbreak of level 5 biohazard aboard [station_name()]. All personnel must contain the outbreak.", "Biohazard Alert", ANNOUNCER_OUTBREAK5)
+		status_alarm("biohazard")
```

Comment:
> This should be added to the random event blob so false alarms also update the status displays

This will be easy to solve. Is there no such problem with level 7 Outbreak?